pipeline/src/main/java/BC_DATABASE_RDD_ENTRY.java

```diff
@@ -66,6 +67,10 @@ public BC_DATABASE_RDD_ENTRY( final BC_DATABASE_SETTING a_setting, final BC_CHUN
     {
         setting = a_setting;
         chunk = a_chunk;
+        if ( setting.ballast > 0 )
+            ballast = new byte[ setting.ballast ];
```

Comment:
IIRC we discussed that this "ballast" should be populated, otherwise Spark may compress it and practically nullify its effect.